go.mod

```diff
@@ -33,15 +34,15 @@ require (
 	//
 	// when updating k8s.io/kubernetes, make sure to update the replace section too
 	//
-	k8s.io/api v0.31.3
-	k8s.io/apimachinery v0.31.3
+	k8s.io/api v0.32.0
```

Comment:
This requires changing the `replace` section too. #5015 is filed to make sure everything is working with the new Kubernetes APIs.